src/main/java/frc/robot/poseestimator/Pose2dArrayValue.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.poseestimator;
+
+public enum Pose2dArrayValue {
```

Comment:
consider to rename, for someone that doesn't know the code i deubt it will be clear

Reply:
Converts array of pose2d -> what it means

It's kinda odd usecase but I can't think of a better name